incubator/nginx/nginx.libsonnet

```diff
@@ -0,0 +1,149 @@
+local k = import 'ksonnet.beta.2/k.libsonnet';
+local deployment = k.extensions.v1beta1.deployment;
+local container = deployment.mixin.spec.template.spec.containersType;
+
+{
+  parts:: {
+    deployment:: {
+      local defaults = {
+        imageTag: "1.10.2-r3",
+        imagePullPolicy: "IfNotPresent",
+      },
+
+      noVhost(namespace, name, labels={app: name})::
+        base(namespace, name, labels),
+
+      hasVhost(namespace, name, labels={app: name})::
```

Comment:
Hmm, what do you think about naming `noVhost` to `simple` and `hasVhost` to `withServerBlock` (nginx has server blocks, not vhosts)? Otherwise, people have to know what a server block/vhost is in order to get started.

Reply:
Sounds good!